config/peerpods/podvm/libvirt-podvm-image-cm.yaml

```diff
@@ -25,4 +25,8 @@ data:
 
   # To Enable SE for IBM Z
   SE_BOOT: "true"
-  
+
+  # For Pre-built PodVM images.
+  LIBVIRT_IMAGE_TYPE: "pre-built"
+  LIBVIRT_IMAGE_SOURCE: "quay.io/openshift_sandboxed_containers/libvirt-podvm-image:latest"
```

Comment:
How about a url like ?
`<repo_type>::<repo_url>::<image_path>`

eg.
PODVM_IMAGE_URI: "oci::quay.io/openshift_sandboxed_containers/libvirt-podvm-image:latest::/image/podvm-390x.qcow2"


Then if we want to support object store, the url can be something like
PODVM_IMAGE_URI: "s3::https://s3-bucket-podvm-image::podvm.raw"